### ISSUE TITLE
TypeError: Cannot read property 'type' of undefined

### ISSUE THREAD

**Issue by arjunnaha** [outline/rich-markdown-editor#186]
When parsing the below post, the following error occurs:

```
checkboxes.ts:59 Uncaught TypeError: Cannot read property 'type' of undefined
    at Array.<anonymous> (checkboxes.ts:59)
    at Core.push../node_modules/markdown-it/lib/parser_core.js.Core.process (parser_core.js:51)
    at MarkdownIt.push../node_modules/markdown-it/lib/index.js.MarkdownIt.parse (index.js:523)
    at MarkdownParser.parse (from_markdown.js:211)
    at RichMarkdownEditor.createDocument (index.tsx:343)
    at RichMarkdownEditor.createState (index.tsx:324)
    at RichMarkdownEditor.createView (index.tsx:348)
    at RichMarkdownEditor.init (index.tsx:182)
    at RichMarkdownEditor.componentDidMount (index.tsx:139)
...
```

The post in question, slightly redacted (but happy to share over email):
```markdown
![](imageurl)\n\n#### **[12/05/2020 Session Plan](url)**\n\n**[14/05/2020 Session Plan](url)**\n\n#### [16/05/2020 Session Plan](url)\n\n#### **[Tactical Master Class - Claude Henderson T20 Challenge](url)**\n\n#### **[Tops Tips from Claude Henderson](url)**\n\n**[Suggested](url) Podcast this week**\n\n[The Cricket Show: Joe Root on how Test cricket can return](https://play.acast.com/s/skysportscricket/https%3A%2F%2Fapi.spreaker.com%2Fepisode%2F26870913)\n\n[ ](https://play.acast.com/s/skysportscricket/https%3A%2F%2Fapi.spreaker.com%2Fepisode%2F26870913)\n\n**[Spin Masterclass session](url)**\n\n\\\n
```
I can't seem to get that markdown to render, but it was produced by the editor, and saved directly to a DB.

**To Reproduce**
1. Set the default value of the editor to the post above
2. Editor throws error

**Expected behavior**
The editor should render the post.

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome
 - Version: 81

**Comment by arjunnaha** [comment]
It looks like it's matching a link with ' ' as the text, as a checkbox:

![image](https://user-images.githubusercontent.com/7504589/81716680-8f9f0480-9471-11ea-9817-e8ff98cb8c6e.png)

**Comment by tommoor** [comment]
Why did you close?

**Comment by arjunnaha** [comment]
Sorry, I referenced this in a PR and closed that, which closed this linked issue - my mistake.

**Comment by tommoor** [comment]
closed by eed39d6a85a7e241c6aa169f3b86cd871cfc55ec
